app/javascript/spec/remove-catalog-item-modal/remove-catalog-item-modal.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import fetchMock from 'fetch-mock';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import RemoveCatalogItemModal from '../../components/remove-catalog-item-modal';
+
+describe('RemoveCatalogItemModal', () => {
+  const item1 = 123;
+  const item2 = 456;
+  const url1 = `/api/service_templates/${item1}?attributes=services`;
+  const url2 = `/api/service_templates/${item2}?attributes=services`;
+  const apiResponse1 = {id: item1, name: 'name123', service_type: 'atomic', services: []};
+  const apiResponse2 = {id: item2, name: 'name456', service_type: 'atomic', services: []};
+  const store = configureStore()({});
+  const dispatchMock = jest.spyOn(store, 'dispatch');
+
+  afterEach(() => {
+    fetchMock.reset();
+    fetchMock.restore();
```

Comment:
You need just one of these. Restore is just an alias for restore.

Reply:
Fixed.